Sources/StreamChatUI/Components.swift

```diff
@@ -311,6 +311,9 @@ public struct Components {
 
     /// The view that displays the number of unread messages in the chat.
     public var messageHeaderDecorationView: ChatChannelMessageHeaderDecoratorView.Type = ChatChannelMessageHeaderDecoratorView.self
+    
+    /// A flag which determines if download action is shown for file attachments.
+    public var isDownloadFileAttachmentsEnabled = true
```

Comment:
Should be false by default:
```suggestion
    public var isDownloadFileAttachmentsEnabled = false
```

Reply:
It is false by default now.